### ISSUE TITLE
Sitemaps are killing us again (This time it's the RECAP sitemaps)

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#855]
Gotta look at this again. I'm not sure the solution, but I'm seeing query logs like this:

```
363306361 [qtp763309998-5180] INFO  org.apache.solr.core.SolrCore  – [recap] webapp=/solr path=/select/ params={caller=r_sitemap&sort=dateFiled+asc&group.ngroups=true&fl=absolute_url,docket_absolute_url,local_path,timestamp&start=1078750&q=*&group.field=docket_id&group=true&wt=json&rows=250} hits=35203230 status=0 QTime=189689 

363308846 [qtp763309998-5204] INFO  org.apache.solr.core.SolrCore  – [recap] webapp=/solr path=/select/ params={caller=r_sitemap&sort=dateFiled+asc&group.ngroups=true&fl=absolute_url,docket_absolute_url,local_path,timestamp&start=449000&q=*&group.field=docket_id&group=true&wt=json&rows=250} hits=35203230 status=0 QTime=186328 
```

Note the query time at the end of about three minutes. These are dragging down user queries. I'm disabling them for now.

Not sure the solution. We've been dealing with this a LONG time. I think the problem is that no matter how you cut the cake, you have to sort 35M items before you can return a result, and that's just not possible.

So...ideas:

 - Smaller result set? Right now it's 250 items, but I'm pretty sure the issue is sorting.

 - Aggressive caching? We played with this before, but it still sucks because of cache invalidation.

Kinda stumped really. This is always the issue that comes up when we scale to the next level.

**Comment by mlissner** [comment]
Well, I added a cache, and that's progress I guess.

**Comment by mlissner** [comment]
Should be fixed in #869.
